tests/integration/test_merge_tree_s3/test.py

```diff
@@ -333,3 +333,26 @@ def test_move_replace_partition_to_another_table(cluster):
 
     for obj in list(minio.list_objects(cluster.minio_bucket, 'data/')):
         minio.remove_object(cluster.minio_bucket, obj.object_name)
+
+
+def test_freeze_unfreeze(cluster):
+    create_table(cluster, "s3_test")
+
+    node = cluster.instances["node"]
+    minio = cluster.minio_client
+
+    node.query("INSERT INTO s3_test VALUES {}".format(generate_values('2020-01-03', 4096)))
+    assert len(
+        list(minio.list_objects(cluster.minio_bucket, 'data/'))) == FILES_OVERHEAD + FILES_OVERHEAD_PER_PART_WIDE
+
+    node.query("ALTER TABLE s3_test FREEZE WITH NAME 'backup1'")
+    assert len(
+        list(minio.list_objects(cluster.minio_bucket, 'data/'))) == FILES_OVERHEAD + FILES_OVERHEAD_PER_PART_WIDE
+
+    node.query("TRUNCATE TABLE s3_test")
+    assert len(
+        list(minio.list_objects(cluster.minio_bucket, 'data/'))) == FILES_OVERHEAD + FILES_OVERHEAD_PER_PART_WIDE
+
+    node.query("ALTER TABLE s3_test UNFREEZE 'backup1'")
```

Comment:
This command looks inconsistent.
Maybe better to make it look `UNFREEZE WITH NAME '...'`?
And also add `UNFREEZE PARTITION`?